site/docs/flink.md

```diff
@@ -78,12 +81,25 @@ as the following:
 # HADOOP_HOME is your hadoop root directory after unpack the binary package.
 export HADOOP_CLASSPATH=`$HADOOP_HOME/bin/hadoop classpath`
 
+# download Iceberg dependency
+ICEBERG_VERSION=0.11.1
+MAVEN_URL=https://repo1.maven.org/maven2
+ICEBERG_MAVEN_URL=$MAVEN_URL/org/apache/iceberg
+ICEBERG_PACKAGE=iceberg-flink-runtime
+wget $ICEBERG_MAVEN_URL/$ICEBERG_PACKAGE/$ICEBERG_VERSION/$ICEBERG_PACKAGE-$ICEBERG_VERSION.jar
+
 # wget the flink-sql-connector-hive-2.3.6_2.11-1.11.0.jar from the above bundled jar URL firstly.
+HIVE_VERSION=2.3.6
+SCALA_VERSION=2.11
+FLINK_VERSION=1.11.0
+FLINK_MAVEN_URL=$MAVEN_URL/org/apache/flink
+FLINK_CONNECTOR_PACKAGE=flink-sql-connector-hive
+wget $FLINK_MAVEN_URL/$FLINK_CONNECTOR_PACKAGE-$HIVE_VERSION\_$SCALA_VERSION/$FLINK_VERSION/$FLINK_CONNECTOR_PACKAGE-$HIVE_VERSION\_$SCALA_VERSION-$FLINK_VERSION.jar
```

Comment:
Nit: Why is the `_` before `$SCALA_VERSION` escaped with a backslash?

If it's to ensure that environment variables are properly expanded, would it be possible to use `${}` to specify the environment variables to avoid having to add the backslash prior to the underscore? Something like this might work: `${FLINK_CONNETO_PACKAGE}-${HIVE_VERSION}_${SCALA_VERSION}/${FLINK_VERSION}`.

For me, the backslash prior to the underscore makes this much harder to read.

Reply:
Thanks for your suggestion. Rework Done